.github/workflows/promote-to-stable.yml

```diff
@@ -0,0 +1,25 @@
+name: Promote
```

Comment:
Not sure, if we should promote to stable via GitHub, since we don't have proper regression tests to do this automatically.

Reply:
The promotion would only happen by one of the maintainers issuing a slash command - and after the app is launched and automatically screenshotted to ensure it launches

Reply:
See here for an example: https://github.com/snapcrafters/signal-desktop/issues/269